valhalla/midgard/logging.h

```diff
@@ -121,31 +121,41 @@ void Configure(const LoggingConfig& config);
 #else
 #ifdef LOGGING_LEVEL_ERROR
 #define LOG_ERROR(x)                                                                                 \
-  ::valhalla::midgard::logging::GetLogger().Log(x, ::valhalla::midgard::logging::LogLevel::ERROR)
+  ::valhalla::midgard::logging::GetLogger().Log(std::string(__FILE__) + ": " +                       \
```

Comment:
woah this is a huge change... I think for sure its ok for LOG_ERROR but i really dont know if we should be doing it for any other log level. im on the fence about WARN..

remember a lot of the logs are informational, showing the line number is not desirable. perhaps you can make another macro for these like `LOGLN_ERROR` etc so its clear you are choosing the macro with the line number in it.

Reply:
Applied your suggestion of `LOGLN_xxx`